packages/amplify-category-storage/src/provider-utils/awscloudformation/default-values/s3-defaults.ts

```diff
@@ -1,6 +1,6 @@
-const uuid = require('uuid');
+import uuid from 'uuid';
 
-const getAllDefaults = project => {
+const getAllDefaults = (project: any) => {
```

Comment:
I recommend reverting these changes for now. I completely changed this file, including converting it to TS in the headless branch: https://github.com/aws-amplify/amplify-cli/tree/headless-storage-s3